requirements/reqs-data.js

```diff
@@ -0,0 +1,266 @@
+exports.requirements = {
```

Comment:
Make this file json instead?

Reply:
Good idea. Fixed that. Gonne commit soon